executors/executor.go

```diff
@@ -9,7 +9,9 @@
 
 package executors
 
-import "encoding/xml"
+import (
+	"encoding/xml"
+)
```

Comment:
unrelated reformatting. either put it in a separate commit if there is a good reason, or otherwise skip it, but don't sneak it in.